pkg/apis/batch/validation/validation.go

```diff
@@ -603,4 +611,6 @@ type JobValidationOptions struct {
 	AllowMutableSchedulingDirectives bool
 	// Allow elastic indexed jobs
 	AllowElasticIndexedJobs bool
+	// Allow Job to have the label on batch.kubernetes.io/job-name and batch.kubernetes.io/controller-uid
+	AllowBatchLabel bool
```

Comment:
Here you call it "allow" implying it's optional, but `ValidateGeneratedSelector` is really _requiring_ values.

Can we rename this to `RequirePrefixedLabels` or something like that?

Reply:
done.